gatsby-node.js

```diff
@@ -144,6 +158,15 @@ exports.modifyWebpackConfig = ({ config, stage }) => {
     current.loader = 'html-loader'
     return current
   })
+  config.loader('handlebars-loader', function(current) {
```

Comment:
It might be out of the scope for this PR, and this was originally something I suggested, but another way we might do this is to leverage gatsby's createPage and createNode similar to how we are doing the markdown pages. We would need to write our own transformer, but it looks like node's handlebars package is pretty straightforward. This would allow people to not have to actually create an index.js file.  Demos would be able to just live under demos, with only a handlebars file.  Example
```
File Structure
-src/
--demos/
----dashboard.hbs
--templates/
----DemoPage.js //handles loading html from hbs file
```
This would allow contributors to only focus on writing the HTML, CSS, and Handlebars. The loader seems to work for now so I would say we only need to do this if others do not like this setup.  it also might simplify the directory layout.